### ISSUE TITLE
Removing the space between an h level and the next paragraph

### ISSUE THREAD

**Issue by suzanneIRB** [wet-boew/wet-boew-styleguide#248]
We are looking for a class which would remove the space between two paragraphs and or an H level and a paragraph or div. rather than using a <br>

On our website we use the class tight

**Comment by suzanneIRB** [comment]
sorry. I continue...    .tight {margin-top:-6px;}
Here's an example of a page where the legend that follows the image would contain the class tight. http://www.irb-cisr.gc.ca/Eng/BoaCom/empl/Pages/MemComMspPsc.aspx

**Comment by suzanneIRB** [comment]
The class tight is attached to a <p> tag preceded by an H2 and removes the space between the H2 and the P.

**Comment by thomasgohard** [comment]
You can use the `mrgn-{tp|bttm}-0` classes for this.

**Comment by suzanneIRB** [comment]
Thank you. I have copied this class to our stylesheet as we are still using WET 3.  I tried it and it made no change, the space was still there. I will try it again.

In the meantime, would there a separate javascript that makes it work that wouldn’t necessarily be part of the WET 3 template, hence the reason why it is not working?



Bonne journée / Have a nice day
Suzanne Burbidge

Agente de soutien du Web
Commission de l'immigration et du statut de réfugié du Canada / Gouvernement du Canada
suzanne.burbidge@cisr-irb.gc.ca<mailto:suzanne.burbidge@cisr-irb.gc.ca>
Tél. lundi, mercredi & vendredi: 613-769-9006 • mardi & jeudi : 613-834-9957
Télécopieur : 613-996-0270
Twitter : @CISR_Canada

Web Support Officer
Immigration and Refugee Board of Canada / Government of Canada
suzanne.burbidge@irb-cisr.gc.ca<mailto:suzanne.burbidge@irb-cisr.gc.ca>
Tel. Monday, Wednesday & Friday: 613-769-9006 • Tuesday & Thursday: 613-834-9957
Fax: 613-996-0270
Twitter: @IRB_Canada
[cid:image001.jpg@01D0A1F4.684292F0]

**Comment by thomasgohard** [comment]
In WET 3, the equivalent classes are called `margin-{top|bottom}-none`.

**Comment by suzanneIRB** [comment]
Yes, however, we are changing all of our wet3 and in-house classes in our existing documents to reflect the ones in WET4. We are copying to our stylesheets, their equivalent from WET4.

. mrgn-tp-0 {margin-top:0;}, does not work nor does margin-top-0 in WET3 for that matter, which is why we had to create our own class to remove the space.

In order for this to work, the value needs to be negative, as in our in-house class.

.tight {margin-top:-6px;}
So is there a class in WET4 to address this scenario?

Bonne journée / Have a nice day
Suzanne Burbidge

Agente de soutien du Web
Commission de l'immigration et du statut de réfugié du Canada / Gouvernement du Canada
suzanne.burbidge@cisr-irb.gc.ca<mailto:suzanne.burbidge@cisr-irb.gc.ca>
Tél. lundi, mercredi & vendredi: 613-769-9006 • mardi & jeudi : 613-834-9957
Télécopieur : 613-996-0270
Twitter : @CISR_Canada

Web Support Officer
Immigration and Refugee Board of Canada / Government of Canada
suzanne.burbidge@irb-cisr.gc.ca<mailto:suzanne.burbidge@irb-cisr.gc.ca>
Tel. Monday, Wednesday & Friday: 613-769-9006 • Tuesday & Thursday: 613-834-9957
Fax: 613-996-0270
Twitter: @IRB_Canada
[cid:image001.jpg@01D0A28E.0A33D850]

**Comment by thomasgohard** [comment]
In the example you provided, there's a `margin-bottom` on the `h2` and a `margin-top` on the `p`. To simulate your class `tight`, you would apply `margin-bottom-none`/`mrgn-bttm-0` on the `h2` and `margin-top-none`/`mrgn-tp-0` on the `p`.
